foodnet/membership/factories.py

```diff
@@ -0,0 +1,58 @@
+import factory
+
+
+class UserFactory(factory.DjangoModelFactory):
+    username = factory.Sequence(lambda n: 'user%d@food.net' % n)
```

Comment:
well I think currently we use uuid4 for this so is there a strong need to do it this way?